core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -262,14 +262,19 @@ public String getMountPoint(AlluxioURI uri) throws InvalidPathException {
 
   /**
    * @param uri the Alluxio uri to check
-   * @return true if the specified uri is mount point, or has a descendant which is a mount point
+   * @param containsSelf cause method to return true when given uri itself is a mount point
+   * @return true if the given uri has a descendant which is a mount point [, or is a mount point]
    */
-  public boolean containsMountPoint(AlluxioURI uri) throws InvalidPathException {
+  public boolean containsMountPoint(AlluxioURI uri, boolean containsSelf)
```

Comment:
can we add some unittests to `MountTableTest` for this method?

Reply:
MountTableTest#path does testing of reverseResolve. That's why I deleted old dupicates.

Reply:
Are there any test cases for `containsMountPoint`?

Reply:
Added into `MountTableTest#pathNestedMount`.